client/webserver/http.go

```diff
@@ -83,16 +83,19 @@ func (s *WebServer) handleLogin(w http.ResponseWriter, r *http.Request) {
 // registerTmplData is template data for the /register page.
 type registerTmplData struct {
 	CommonArguments
-	Initialized    bool
-	Authed         bool
 	KnownExchanges []string
+	// Host is optional. If provided, the register page will not display the add
+	// dex form, instead this host will be pre-selected for registration.
+	Host string
```

Comment:
If #2025 goes in first (I think it should), I'd use the dexsettings page for this instead of the register page. It would already contain the mechanics for posting bond for an existing dex connection and would naturally fit.